docs/zh/latest/config.json

```diff
@@ -86,7 +92,17 @@
             "plugins/kafka-logger",
             "plugins/udp-logger",
             "plugins/syslog",
-            "plugins/log-rotate"
+            "plugins/log-rotate",
+            "plugins/error-log-logger",
+            "plugins/sls-logger"
+          ]
+        },
+        {
+          "type": "category",
+          "label": "Other Protocols",
```

Comment:
```suggestion
          "label": "其它",
```

Reply:
@juzhiyuan should I commit these changes?

Reply:
Let me help you